apps/system/js/popup_window.js

```diff
@@ -21,6 +21,19 @@
     };
 
     AppWindow.call(this, configs);
+
+    // Replicate the theme color from the parent window.
+    // See http://bugzil.la/1132418
+    if (this.rearWindow) {
```

Comment:
A few lines above, the existence of `configs.rearWindow` is tested, so I thought it makes sense to test `this.rearWindow` too. But I can remove it if you think it's useless.